recipes-security/optee/optee-test-tegra_35.1.0.bb

```diff
@@ -0,0 +1,45 @@
+SUMMARY = "OP-TEE sanity testsuite"
+DESCRIPTION = "Open Portable Trusted Execution Environment - Test suite"
+HOMEPAGE = "https://www.op-tee.org/"
+
+LICENSE = "BSD-2-Clause & GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=daa2bcccc666345ab8940aab1315a4fa"
+
+inherit python3native
+require optee-tegra.inc
+
+DEPENDS = "python3-cryptography-native optee-os-tadevkit-tegra optee-client-tegra"
+
+S = "${WORKDIR}/optee/optee_test"
+B = "${WORKDIR}/build"
+
+EXTRA_OEMAKE += " \
+    LIBGCC_LOCATE_CFLAGS='--sysroot=${STAGING_DIR_HOST}' \
+    CROSS_COMPILE='${HOST_PREFIX}' \
+    PYTHON3='${PYTHON}' \
+    TA_DEV_KIT_DIR='${TA_DEV_KIT_DIR}' \
+    OPTEE_CLIENT_EXPORT='${STAGING_DIR_HOST}/usr' \
+    O='${B}' \
+"
+
+do_compile() {
+    oe_runmake -C ${S} all
+}
+do_compile[cleandirs] = "${B}"
+
+do_install () {
+    install -d ${D}${bindir}/xtest
```

Comment:
Remove this.  The upstream (meta-arm) recipe installs `xtest` directly into `/usr/bin`, and we should follow that.